zjit/src/hir.rs

```diff
@@ -1457,6 +1477,7 @@ impl Function {
             &ArrayMax { ref elements, state } => ArrayMax { elements: find_vec!(elements), state: find!(state) },
             &SetGlobal { id, val, state } => SetGlobal { id, val: find!(val), state },
             &GetIvar { self_val, id, state } => GetIvar { self_val: find!(self_val), id, state },
+            LoadPC => LoadPC,
```

Comment:
This could go up top in the shallow clone case

Reply:
:+1: https://github.com/ruby/ruby/pull/14643/commits/1b6528aa09ded8e7575287994e52351f8b50783e